apps/e2e/integration/reconciliation_order.spec.ts

```diff
@@ -1621,6 +1621,247 @@ testOrders("commit: applies delivery updates to customer order lines", async ({
 	await dialog.getByRole("button", { name: "Confirm" }).click();
 	await dialog.waitFor({ state: "detached" });
 
+	//more assertions to give time for the line to be updated to delivered
+
+	// navigate to customer order view
+	await page.getByRole("navigation").getByRole("listitem").nth(5).click();
```

Comment:
Why not `getByRole("link", { name:  "Customers" })`?

Reply:
I don't remember why but it's not working